audb/core/publish.py

```diff
@@ -28,7 +28,10 @@ def _check_for_duplicates(
     def job(table):
         audformat.assert_no_duplicates(table._df)
 
-    tables = db.tables.values()
+    tables = (
```

Comment:
Maybe simplify to `table_ids = list(db)` and then reolace `table._df` with `db[table_id]._df` in `job()`?

Reply:
Good idea, I was anyway not happy with this single cases were I wasn't using `list(db)`. I changed it now accordingly.